docs/requirement-sets/common/identity-api-requirement-sets.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: Identity API requirement sets
 description: Identity API requirement set information for Office Add-ins.
-ms.date: 09/28/2022
+ms.date: 07/07/2023
 ms.topic: overview
+ms.prod: non-product-specific
+
```

Comment:
No more need for ms.prod. We now use ms.service and it's handled in the [docfx.json](https://github.com/OfficeDev/office-js-docs-reference/blob/main/docs/docfx.json) unless you want to override its value at the file level. The default ms.service value for this file found in the docfx is microsoft-365.

```suggestion
```